lib/logstash/outputs/bson/big_decimal.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2009-2014 MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Modified 2015 Elastic
+
+module BSON
+
+  # Injects behaviour for encoding and decoding BigDecimal values
+  # to and from # raw bytes as specified by the BSON spec.
+  #
+  # @see http://bsonspec.org/#/specification
+  module BigDecimal
+
+    # A floating point is type 0x01 in the BSON spec.
+    BSON_TYPE = 1.chr.force_encoding(BINARY).freeze
+
+    # The pack directive is for 8 byte floating points.
+    PACK = "E".freeze
```

Comment:
is there any reason to use little-endian format here? I mean not saying is wrong, just asking to know why :smile_cat: